src/one_wire.toit

```diff
@@ -31,13 +31,22 @@ class Bus:
   */
   static SKIP_FAMILY ::= Object
 
-  protocol_/Protocol? := ?
+  /**
+  The protocol.
+  Use the $protocol_ accessor to use the protocol.
+  */
+  protocol__/Protocol? := ?
+
+  protocol_ --power/bool [block] -> none:
+    if not protocol__: throw "BUS CLOSED"
+    protocol__.set_power power
+    block.call protocol__
```

Comment:
I don't understand the benefit of the block.  Nothing is done after the block.  Why not just return the protocol_ ?

Reply:
Agreed. It's easier to just return the protocol.
done.